### ISSUE TITLE
[bug]: Emails are not sending for correct SMTP configurations

### ISSUE THREAD

**Issue by ibtesamlatif2997** [makeplane/plane#4439]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

Last week I have installed the self hosted docker version and successfully setup admin and workspace. From god mode I have setup the SMTP configurations and it's saving successfully, but still emails are not being sending. The same SMTP configurations are working from other platforms

I am using aws ses service for sending emails

The emails should be sent from plane, if the same configurations are working from other places

### Steps to reproduce

1. Go to god mode
2. Setup smtp configurations
3. Invite people to workspace or signup from authentication code
4. The emails are not receving

### Environment

Production

### Browser

Google Chrome

### Variant

Self-hosted

### Version

stable

**Comment by ReggieMarr** [comment]
I have the same issue.
I think it has to do with smtp verification not allowed (aka using an email service that requires 2FA).

I tried running the following:
```
➜  plane-app docker container ls
CONTAINER ID   IMAGE                             COMMAND                  CREATED          STATUS          PORTS                                   NAMES
2ab42776044b   makeplane/plane-proxy:stable      "/docker-entrypoint.…"   19 minutes ago   Up 19 minutes   0.0.0.0:8855->80/tcp, :::8855->80/tcp   plane-app-proxy-1
65be6cc2621f   makeplane/plane-space:stable      "docker-entrypoint.s…"   19 minutes ago   Up 19 minutes   3000/tcp                                plane-app-space-1
0fc0eacbe79d   makeplane/plane-frontend:stable   "docker-entrypoint.s…"   19 minutes ago   Up 19 minutes   3000/tcp                                plane-app-web-1
d0df019160f7   makeplane/plane-backend:stable    "./bin/beat"             19 minutes ago   Up 19 minutes   8000/tcp                                plane-app-beat-worker-1
ef70f5708846   makeplane/plane-backend:stable    "./bin/worker"           19 minutes ago   Up 19 minutes   8000/tcp                                plane-app-worker-1
6b2d0644acee   makeplane/plane-backend:stable    "./bin/takeoff"          19 minutes ago   Up 19 minutes   8000/tcp                                plane-app-api-1
acc2c69b7d45   redis:7.2.4-alpine                "docker-entrypoint.s…"   19 minutes ago   Up 19 minutes   6379/tcp                                plane-app-plane-redis-1
30e9568d330f   minio/minio                       "/usr/bin/docker-ent…"   19 minutes ago   Up 19 minutes   9000/tcp                                plane-app-plane-minio-1
8c49d3860cca   postgres:15.5-alpine              "docker-entrypoint.s…"   19 minutes ago   Up 19 minutes   5432/tcp                                plane-app-plane-db-1

➜  plane-app docker exec -it ef70f5708846 /bin/sh
/code # python3 manage.py test_email [user email]
Trying to send test email...
Error: Email could not be delivered due to (535, b'5.7.139 Authentication unsuccessful, SmtpClientAuthentication is disabled for the Tenant. Visit https://aka.ms/smtp_auth_disabled for more information. [YT1PR01CA0140.CANPRD01.PROD.OUTLOOK.COM 2024-05-15T16:11:37.399Z 08DC4B03C00638EE]')
```

**Comment by mologie** [comment]
Same issue on our test instance, but it is not related to SMTP authentication config at all.

The test command succeeds at sending, and the email is received:
```
/code # ./manage.py test_email snip@example.com
Trying to send test email...
Email successfully sent
```

Users also get invite emails. However, after accepting the invite, **the email login code on the login page is never delivered.** (It is unclear to me why users can't set a password right away after invite either, seems counter-intuitive to have two mails after another.)

**Comment by pushya22** [comment]
@ReggieMarr @mologie Are you still facing the issue?

**Comment by ReggieMarr** [comment]
Yeah I found it was something to do with outlook, ending up working around with a a gmail account.

**Comment by BANG88** [comment]
‌‌‌Has anyone used resend to send emails? The same configuration works elsewhere, but it doesn't work here on plane.

`Error: Email could not be delivered due to [Errno 101] Network unreachable`

**Comment by mologie** [comment]
I don't know, we stopped demoing Plane after it had too many oddities with email setup. Just to list a few issues we found that contribute to this one:
* Changing email configuration in the admin panel would not apply until Plane was restarted. Refreshing would show the old information. This made stuff hard to debug.
* It was not possible to use an internal SMTP server without authentication. It refused to save at all.
* TLS toggles were inverted and every confusing (they changed text depending on state).
* Emails were not sent to a colleague of mine like described in a comment above. We later switched to AWS SES, same issue.

**Comment by isrealbm** [comment]
Same here. I was tested with SES, Mailtrap, and Sendgrid... all failed.

**Comment by mguptahub** [comment]
@pablohashescobar

**Comment by andreyrd** [comment]
I don't know if you're having the same issue, but mine was because I'm running on Kubernetes, and because #5778 the worker pod was getting killed and never had the chance to send the emails out.

**Comment by muffintime77** [comment]
![изображение](https://github.com/user-attachments/assets/4f06754b-7d14-4001-96a4-d7c1bea2a6eb)
I see a message in the logs when checking. is there any solution to the problem?

**Comment by GaNacereddine** [comment]
> ![изображение](https://private-user-images.githubusercontent.com/57962355/389064894-4f06754b-7d14-4001-96a4-d7c1bea2a6eb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYwODk5ODUsIm5iZiI6MTczNjA4OTY4NSwicGF0aCI6Ii81Nzk2MjM1NS8zODkwNjQ4OTQtNGYwNjc1NGItN2QxNC00MDAxLTk2YTQtZDdjMWJlYTJhNmViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA1VDE1MDgwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzOGIwMjcyZDkyNGIxZDBlZDAyMDVkNWVjODViNWExMzVlZDc5NDM4NWYxZjg4MjFlNmQzNDgyNjdmZjQ4ZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.F0FMflyhIiWh2PV6bOiUAqWJnSUsEGviT9lIM3IZ1hI) I see a message in the logs when checking. is there any solution to the problem?

I just changed the option to use without email security and it worked

**Comment by Fink-Git** [comment]
I have the same issue when i'm testing email config in god-mode on self-hosting instance.

My SMTP spec : 
- No email security
- No authentication

I have this message 
"Could not send email. Please check your configuration"

And when i look into API logs :
```
api-1  | WARNING 2025-01-09 15:44:16,594 log 25 140229855030072 Bad Request: /api/instances/email-credentials-check/
api-1  | XXXXX - "POST /api/instances/email-credentials-check/ HTTP/1.1" 400
```

**Comment by dassels** [comment]
I'm having the exact same issue in the latest version. I hope there is a fix soon, coming up on one year for this SMTP problem and it's really not usable without getting email notifications from the app.

**Comment by akshat5302** [comment]
Closing this as stale: there hasn't been any activity here for a while, and it was reported against a release that is well behind current Plane (Community v1.4.2 / Commercial v3.2.0).

If you still hit this on a current version, please open a new issue with your Plane version, install method and container logs, and link back here. Thanks!
